src/main/java/net/sf/rails/ui/swing/elements/Spinner.java

```diff
@@ -11,8 +11,7 @@
     private Color buttonColour = new Color(255, 220, 150);
 
     public Spinner(int value, int from, int to, int step) {
-        super(new SpinnerNumberModel(new Integer(value), new Integer(from),
-                to > 0 ? new Integer(to) : null, new Integer(step)));
+        super(new SpinnerNumberModel(Integer.valueOf(value), Integer.valueOf(from), to > 0 ? to : null, Integer.valueOf(step)));
```

Comment:
Autoboxing?

Reply:
Yep. I'm sure there are lots of changes like this that still need to be made!

Reply:
Actually I had looked at this one-- can't change it because the autoboxing to `Integer` triggers the `SpinnerNumberModel(Number value, Comparable<?> minimum, Comparable<?> maximum, Number stepSize)` constructor vs the `SpinnerNumberModel(int value, int minimum, int maximum, int stepSize)` constructor.